prow/config/config.yaml

```
@@ -1487,9 +1487,12 @@ branch-protection:
               required_status_checks:
                 contexts:
                   - "DCO"
-                  - "go (build)"
-                  - "go (test)"
-                  - "go (verify)"
+                  - "ci / go (amd64, build)"
+                  - "ci / go (amd64, test)"
+                  - "ci / go (amd64, verify)"
+                  - "ci / go (arm64, build)"
+                  - "ci / go (arm64, test)"
+                  - "ci / go (arm64, verify)"
```

Remove the prefix and suffix of GitHub Action.

```suggestion
                  - "go (amd64, build)"
                  - "go (amd64, test)"
                  - "go (amd64, verify)"
                  - "go (arm64, build)"
                  - "go (arm64, test)"
                  - "go (arm64, verify)"
```

---

I don't think so. It seems that it doesn't work well without the prefix. See https://github.com/chaos-mesh/chaosd/pull/170

---

@YangKeao, Because [this PR](https://github.com/chaos-mesh/chaosd/pull/170) does not submit a new commit to trigger a new CI. 

See: https://github.com/chaos-mesh/chaosd/pull/158